src/main/groovy/com/epam/petclinic/plugin/QualityAwareJavaPlugin.groovy

```diff
@@ -49,24 +51,29 @@ public class QualityAwareJavaPlugin implements Plugin<Project> {
             ]
         }
 
-        configureCheckStyle(project)
-        configurePMD(project)
-        configureFindBugs(project)
+        project.afterEvaluate {
+            configureCheckStyle(project)
+            configurePMD(project)
+            configureFindBugs(project)
+        }
     }
 
     private void configureCheckStyle(Project project) {
         project.plugins.apply(CHECKSTYLE_PLUGIN_ID)
 
         project.checkstyle {
-            toolVersion = "7.1"
-            config = getToolResource(project, "checkstyle/checkstyle-rules.xml")
-            configProperties.suppressionsFile = getToolPath(project, "checkstyle/checkstyle-suppressions.xml")
+            toolVersion = project.javaQuality.checkstyleToolVersion
+            config = getToolResource(project, 'checkstyle/checkstyle-rules.xml')
+            configProperties.suppressionsFile =
```

Comment:
What do you mean? 
`Checkstyle` and `Pmd` plugins have different extensions and can't be configured in the same way